sqlparser/lib/src/reader/tokenizer/token.dart

```diff
@@ -29,14 +29,20 @@ enum TokenType {
   ampersand,
   analyze,
   and,
+  any,
```

Comment:
We should probably make these postgres-specific (or at least the `keyword` mapping below). I can take care of adding the necessary logic in the tokenizer since it will probably be a bit ugly. But are we using these tokens anywhere?

Reply:
These only for quoting. Drift use quotes when one of this keywords are used. I don't think it's need to be separated for Postgres.

Reply:
>  I don't think it's need to be separated for Postgres.

The thing is that tests are now failing because the parser is overly strict and always interprets keywords as keywords. So doing `SELECT any FROM table` would be valid in sqlite3 (selects a column called "any"), but the parser now complains that  "any" is an unexpected keyword here.

The list of keywords used for escaping at runtime is available [here](https://github.com/simolus3/moor/blob/develop/drift/lib/sqlite_keywords.dart), that one can be changed without issues :D